### PR TITLE
[Refactor]: Move fmtutil from cmd to pkg/formatter

### PR DESCRIPTION
**Pull request by Zheaoli**
Signed-off-by: Zheao.Li <me@manjusaka.me>

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]
Thanks

File: pkg/fmtutil/fmtutil.go [hunk 1]
```diff
@@ -45,13 +45,13 @@ func formatLabels(labels map[string]string) string {
...
-// formatSlice formats the slice with `--format` flag.
+// FormatSlice formats the slice with `--format` flag.
 //
 // --format="" (default): JSON
 // --format='{{json .}}': JSON lines
 //
-// formatSlice is expected to be only used for `nerdctl OBJECT inspect` commands.
-func formatSlice(cmd *cobra.Command, x []interface{}) error {
+// FormatSlice is expected to be only used for `nerdctl OBJECT inspect` commands.
+func FormatSlice(cmd *cobra.Command, x []interface{}) error {
```
Comment (AkihiroSuda) [inline pkg/fmtutil/fmtutil.go:54]:
```suggestion
func FormatSlice(format string, x []interface{}) error {
```

to decouple `pkg/...` from CLI, or just retain this function under `cmd/nerdctl`

Comment (Zheaoli) [reply pkg/fmtutil/fmtutil.go]: Done(

**Comment by djdongjin** [comment]
Should we combine this `pkg/fmtutil` with `pkg/formatter` package? From the name they are quite similar.

https://github.com/containerd/nerdctl/blob/49aa8fbddeae6f92deaec08178673a9e7c55f661/pkg/formatter/

**Review by djdongjin — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
> pkg/fmtutil

pkg/formatter

**Review by fahedouch — APPROVED** [review]
Thanks

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**